crates/cxx-qt-lib/src/types/qvariant/mod.rs

```diff
@@ -105,7 +105,10 @@ impl QVariant {
     /// Returns the stored value converted to the template type T
     ///
     /// If the value cannot be converted, a default-constructed value will be returned.
-    pub fn value<T: QVariantValue>(&self) -> T {
+    ///
+    /// Note that this calls the `QVariant::value` method, without performance loss.
+    /// Whereas `try_value` first calls `QVariant::canConvert`.
+    pub fn value_or_default<T: QVariantValue>(&self) -> T {
         T::value(self)
```

Comment:
should the trait be renamed as well? Or should that match the Qt API ? :thinking:

Reply:
Renamed otherwise the Rust code looks weird.